ui/components/app/fee-details-component/fee-details-component.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+
+import mockState from '../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import FeeDetailsComponent from './fee-details-component';
+
+jest.mock('../../../store/actions', () => ({
+  getGasFeeEstimatesAndStartPolling: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve()),
+  addPollingTokenToAppState: jest.fn(),
+}));
+
+const render = (state = {}) => {
+  const store = configureStore()({ ...mockState, ...state });
+  return renderWithProvider(<FeeDetailsComponent />, store);
+};
+
+describe('FeeDetailsComponent', () => {
+  it('should render correctly', () => {
```

Comment:
```suggestion
  it('renders "Fee details"', () => {
```

I've been having to remind myself that the contributor docs have requested us not to prepend tests with "should" too 

ref: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test
> Do not use "should" at the beginning of the test name. The official Jest documentation [omits this word from their examples](https://jestjs.io/docs/next/getting-started), and it creates noise when reviewing the list of tests printed after a run.

this would also apply to the other tests below